package.json

```diff
@@ -39,12 +39,12 @@
     "dist"
   ],
   "peerDependencies": {
-    "vue": "^2.7.16"
+    "vue": "3.4.23"
```

Comment:
```suggestion
    "vue": "^3.4.23"
```

Reply:
Agree, it should not be the exact version